### PR TITLE
[15.0][MIG] sales_team_security_sale: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36547

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You can rebase now

File: sales_team_security_sale/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Iván Todorovich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale documents permissions by teams",
+    "summary": "Integrates sales_team_security with sale",
+    "version": "15.0.3.0.0",
```
Comment (pedrobaeza) [inline sales_team_security_sale/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

File: sales_team_security_sale/security/security.xml [hunk 2]
```diff
@@ -1,42 +1,39 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<odoo>
-    <data noupdate="1">
-
+<odoo noupdate="1">
```
Comment (pedrobaeza) [inline sales_team_security_sale/security/security.xml:2]: If you do this, the indentation is now incorrect and provokes the rest of the changes. Is that avoidable?

**Review by pedrobaeza — COMMENTED** [review]
And you cash squash commits this way:

Selección_014

**Comment by cesar-tecnativa** [comment]
@chienandalu could you review please?

**Comment by pedrobaeza** [comment]
Tests are red

File: sales_team_security_sale/security/security.xml (hunk 2 above)
Comment (pedrobaeza) [reply sales_team_security_sale/security/security.xml:2]: Don't join both tags as stated in the other twin PR

**Comment by pedrobaeza** [comment]
Tests are failing

**Comment by cesar-tecnativa** [comment]
> Tests are red

test are green

**Comment by cesar-tecnativa** [comment]
@LudLaf could you review this?

File: sales_team_security_sale/security/security.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="sale_order_team_rule" model="ir.rule">
+        <field name="name">Team Orders</field>
+        <field name="model_id" ref="sale.model_sale_order" />
+        <field
+            name="domain_force"
+        >['|', '|', ('user_id', '=', user.id), ('user_id', '=', False), '|', ('team_id', '=', user.sale_team_id.id),
+            ('team_id', '=', False)]
```
Comment (pedrobaeza) [inline sales_team_security_sale/security/security.xml]: Why splitting this (and the others) in 2 lines?

**Review by pedrobaeza — APPROVED** [review]

File: sales_team_security_sale/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -1 +1,7 @@
 Glue module between sales_team_security and sale.
+
+This module adds a new type of "Sale" group called "User: Team Documents Only",
```
Comment (pedrobaeza) [inline sales_team_security_sale/readme/DESCRIPTION.rst]: This is not correct. This module doesn't add this group. It's added by the dependent module. This module adds the permissions to sales documents according such group.

Comment (pedrobaeza) [reply sales_team_security_sale/readme/DESCRIPTION.rst]: @cesar-tecnativa @LudLaf  please finish this

Comment (LudLaf) [reply sales_team_security_sale/readme/DESCRIPTION.rst]: Fixed.
@pedrobaeza please confirm. Thanks

Comment (pedrobaeza) [reply sales_team_security_sale/readme/DESCRIPTION.rst]: OK.

**Pull request merged**